Makefile

```diff
@@ -528,7 +528,7 @@ require:
 	@go version >/dev/null 2>&1 || (echo -e "ERROR: go is required."; exit 1)
 	@which aclocal > /dev/null || (echo -e  "ERROR: automake is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install automake"; exit 1)
 	@which $(LIBTOOL) > /dev/null || (echo -e "ERROR: libtool is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install libtool"; exit 1)
-	@which ninja > /dev/null || (echo -e "ERROR: Ninja is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install ninja"; exit 1)
+	@which ninja > /dev/null || (echo -e "ERROR: Ninja is required. Install with:\n\n    $(POSIX_PKG_MANAGER) install ninja-build"; exit 1)
```

Comment:
This differs by platform - macOS it's `ninja` and on Ubuntu, I think it is `ninja-build`

Reply:
This possibility crossed my mind, but I didn't go the distance to see if it was the case.
I ran into this while using Debian on WSL2.
I'm going to ponder a better solution for this and flag it for non-use for now

Reply:
> This differs by platform - macOS it's `ninja` and on Ubuntu, I think it is `ninja-build`

I added a conditional that addresses both now